devinfo/src/mountinfo/error.rs

```diff
@@ -65,3 +65,17 @@ impl From<semver::Error> for MountInfoError {
         Self::Semver(semver_err)
     }
 }
+
+impl From<MountInfoError> for std::io::Error {
+    fn from(error: MountInfoError) -> Self {
+        use std::io::{Error, ErrorKind};
+
+        match error {
+            MountInfoError::Io(std_io_err) => std_io_err,
+            MountInfoError::InconsistentRead { .. } => Error::new(ErrorKind::Other, error),
```

Comment:
Maybe busy better to explain the inconsistent read? 

```suggestion
            MountInfoError::InconsistentRead { .. } => Error::new(ErrorKind::ResourceBusy, error),
```

Reply:
Hmm, yeah that actually fits better.

Reply:
ResourceBusy is an unstable feature at the moment unfortunately.

Reply:
Damn, maybe InvalidData then

Reply:
Or whichever one you prefer then

Reply:
Sure, using InvalidData. Putting in a TODO for changing it some time in the future.